packages/styles/scss/components/footer/_language-selector.scss

```
@@ -0,0 +1,94 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../temp-carbon-expressive/button/button-expressive';
+@import 'carbon-components/scss/components/combo-box/combo-box';
+@import 'carbon-components/scss/components/list-box/list-box';
+@import 'carbon-components/scss/components/text-input/text-input';
+@import '../../themes/expressive/components/combo-box';
+@import '../../themes/expressive/components/list-box';
+@import '../../themes/expressive/components/text-input';
+
+/// Footer language selector
+/// @access private
+/// @group footer
+
+@mixin language-selector {
+  .#{$prefix}--language-selector__container {
+    @include combo-box;
+    @include listbox;
+    @include text-input;
+    @include combo-box-expressive;
+    @include list-box-expressive;
+    @include text-input-expressive;
+    @include carbon--make-col-ready;
+    @include carbon--make-col(4, 4);
+
+    margin-top: carbon--mini-units(2);
```

Just for consistency, we've been using spacing tokens for margins/paddings

```suggestion
    margin-top: $spacing-05;
```